General/how-do-i-submit-an-executive-briefing-request.md

```diff
@@ -0,0 +1,44 @@
+{{{
+  "title": "How do i submit an Executive Briefing Request",
+  "date": "02-10-2015",
+  "author": "Dana Bowlin",
+  "attachments": [
+  {
+    "file_name": "Attached File",
```

Comment:
Shouldn't these be names something like  `Customer Direct Form` and `EBC Form`? not both named attached File?